lib/traject/dataspace_research_data_config.rb

```diff
@@ -38,6 +38,9 @@
 to_field 'title_tsim', extract_xpath("/item/metadata/key[text()='dcterms.title']/../value")
 to_field 'uri_tesim', extract_xpath("/item/metadata/key[text()='dc.identifier.uri']/../value")
 
+to_field 'collection_id_ssi', extract_xpath('/item/parentCollection/id')
+to_field 'handle_ssi', extract_xpath('/item/handle')
```

Comment:
These two new files are technically not needed but they are useful to figure out where a given dataset came from in DataSpace and troubleshoot it. These fields are visible via the `raw` URL, e.g. http://localhost:3000/catalog/95011/raw